scopesim/effects/psf_utils.py

```diff
@@ -109,9 +109,9 @@ def make_strehl_map_from_table(tbl, pixel_scale=1*u.arcsec):
     return map_hdu
 
 
-def rescale_kernel(image, scale_factor, spline_order=None):
+def rescale_kernel(image, scale_factor, spline_order=None, cmds=None):
     if spline_order is None:
-        spline_order = utils.from_currsys("!SIM.computing.spline_order")
+        spline_order = utils.from_currsys("!SIM.computing.spline_order", cmds=cmds)
```

Comment:
```suggestion

```

Reply:
OK, if we remove this here, then it is not necessary to add `cmds` as an argument to `rescale_kernel()`. I'll update the code in `psf.py` to get the spline order before calling `rescale_kernel()`.

Note that you can comment on multiple lines at a time by "dragging the plus down" next to the lines in the diff view.